presto-docs/src/main/sphinx/functions/binary.rst

```diff
@@ -58,10 +58,18 @@ Binary Functions
 
     Encodes ``real`` in a 32-bit big-endian binary according to IEEE 754 single-precision floating-point format.
 
+.. function:: from_ieee754_32(varbinary) -> real
```

Comment:
Name the parameter `binary` as that matches what we do for other functions in this section.